pychunkedgraph/app/segmentation/common.py

```diff
@@ -1042,9 +1042,15 @@ def handle_get_layer2_graph(table_id, node_id):
     current_app.table_id = table_id
     user_id = str(g.auth_user.get("id", current_app.user_id))
 
+    if "bounds" in request.args:
```

Comment:
don't we have a common function for parsing this argument? if we don't we should